prefect_transform/tasks.py

```diff
@@ -73,43 +56,13 @@ def create_materialization(
         An `MqlQueryStatusResp` object if `run_async` is `True`.
         An `MqlMaterializeResp` object if `run_async` is `False`.
     """
```

Comment:
Perhaps we can add an example how to use this? e.g.
https://github.com/PrefectHQ/prefect-snowflake/blob/main/prefect_snowflake/database.py#L205-L235